api/can_api_v2_definition.py

```diff
@@ -19,6 +19,18 @@ class TestPositivityRatioMethod(GetByValueMixin, enum.Enum):
     OTHER = "other"
 
 
+class MetricSources(GetByValueMixin, enum.Enum):
+    """Metric source."""
```

Comment:
This is a bit broader comment, but I'd like to fill out the docstrings here a bit - since these descriptions will show up in the docs, it will show up in the docs and would be good to give people context about what these are